src/blog/understanding-ecma-part-1.md

```diff
@@ -0,0 +1,232 @@
+---
+title: 'Understanding the ECMAScript spec, part 1'
+author: '[Marja Hölttä](https://twitter.com/marjakh), speculative specification spectator'
+avatars:
+  - marja-holtta
+date: 2020-01-22 13:33:37
+tags:
+  - ECMAScript
+description: 'Tutorial on reading the ECMAScript specification'
+tweet: ''
+---
+
+... where we take a simple function in the spec and try to understand the notation.
+
+## Preface
+
+Even if you know JavaScript, reading its language specification, the ECMAScript spec, can be pretty daunting. At least that's how I felt when I started reading it for the first time. Reading the spec from the start doesn't help either, as the spec is not designed to be read in that order, and notations are not explained before they occur.
```

Comment:
> notations are not explained before they occur

They usually are, AFAICT? However, the spec is so large that you often start reading a particular section without having read the earlier chapters explaining [the notational conventions](https://tc39.es/ecma262/#sec-notational-conventions).

Reply:
Removed this sentence because it doesn't add much value anyway :)